pkg/awsclient/mock/zz_generated.mock_client.go

```diff
@@ -0,0 +1,919 @@
+// Code generated by MockGen. DO NOT EDIT.
```

Comment:
Note to reviewers: This _should_ have simply showed up as a rename of `mock_client.go`. The contents are identical.